### ISSUE TITLE
Evaluate the inclusion of the sundow reference (SR) partitioning method

### ISSUE THREAD

**Issue by gilbertozp** [fluxnet/ONEFlux#10]
The sundown reference partitioning method was part of the FLUXNET2015 release but is not currently part of ONEFlux. Consider options for including it or removing variable from product generation pipeline.

**Comment by darpap** [comment]
I don't think we need to add it. We did in FLUXNET2015 but it should be fully re-developed and optimized. It is part of the different steps we could add.
My preference is to remove from now from the variables.

**Comment by darpap** [comment]
change the title to remove. The method only works in specific conditions and it is not worth to be added back.
